common/autoinstallers/rush-prettier/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "pretty-quick": "3.1.0",
-    "prettier": "2.3.1"
+    "pretty-quick": "3.1.3",
+    "prettier": "2.7.1"
```

Comment:
If we upgrade Prettier, we should re-prettify all the files to make sure nothing has changed.

Reply:
I'll do that in my other PR.

Reply:
I pushed a small commit with this fix